apps/addons/views.py

```diff
@@ -144,7 +145,12 @@ def extension_detail(request, addon):
 
 @mobilized(extension_detail)
 def extension_detail(request, addon):
-    return render(request, 'addons/mobile/details.html', {'addon': addon})
+    ios_user = False
+    print request.META
+    if request.META.get('HTTP_USER_AGENT') in FIREFOX_IOS_USER_AGENTS:
+        ios_user = True
```

Comment:
Can also be written `ios_user = request.META.get('HTTP_USER_AGENT') in FIREFOX_IOS_USER_AGENTS`